xml/System.Net.PeerToPeer.Collaboration/PeerObject.xml

```diff
@@ -306,7 +306,7 @@
       <Docs>
         <param name="obj">The object to test for equality.</param>
         <summary>Compares the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerObject" /> to the specified object.</summary>
-        <returns>Returns <see langword="true" /> if the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerObject" />  
+        <returns><see langword="true" /> if the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerObject" />  
```

Comment:
```suggestion
        <returns><see langword="true" /> if <paramref name="obj" /> is a <see cref="T:System.Net.PeerToPeer.Collaboration.PeerObject" /> instance, and the <see cref="T:System.Net.PeerToPeer.Collaboration.PeerObject" /> instances have matching data; otherwise, <see langword="false" />.  
```